app/TabNavigation.tsx

```diff
@@ -46,11 +45,11 @@ export function TabNavigation(props: { onLogout: () => void }): ReactElement {
             setdisplayTab(TabDisplay.Location);
           }}
         >
-          Location
+          Locations
         </Button>
       </div>
       {displayTab === TabDisplay.Deals && <Deals />}
-      {displayTab === TabDisplay.Location && <Location />}
+      {displayTab === TabDisplay.Location && <Locations />}
```

Comment:
probably the tab display type should also be `Locations`

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/12/commits/3dcfc47e972c4562e3f1f743eccfa0fa0b0f5f5b